index.html

```diff
@@ -38,7 +38,7 @@
     <meta name="theme-color" content="#e30062" />
   </head>
 
-  <body>
+  <body class="index-body">
```

Comment:
No need to give class to body tag. You can directly apply the css to body tag.
```
body {
  
}
```

Reply:
Done